ngraph/core/reference/include/ngraph/runtime/reference/gather_nd.hpp

```diff
@@ -43,7 +43,6 @@ void gather_nd(const T* const params,
                T* const out,
                const Shape& params_shape,
                const Shape& indices_shape,
-               const Shape& out_shape,
```

Comment:
Changed reference API Here. Arm Plugin needs to be changed too.

Reply:
Change reverted.